src/elements/DoubleSlider/DoubleSlider.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div id="DoubleSlider" :style="cssVars">
+    <div id="left-progressBar"></div>
+    <div id="right-progressBar"></div>
+    <input id="faderLeft" type="range" v-model="faderLeftValue"
+      v-bind:min="sliderMin"
+      v-bind:max="sliderMax"
+      v-bind:step="sliderStep"
+      v-bind:aria-valuemin="sliderMin"
+      v-bind:aria-valuemax="sliderMax"
+      v-bind:aria-valuenow="faderLeftValue">
+    <input id="faderRight" type="range" v-model="faderRightValue"
+      v-bind:min="sliderMin"
+      v-bind:max="sliderMax"
+      v-bind:step="sliderStep"
+      v-bind:aria-valuemin="sliderMin"
+      v-bind:aria-valuemax="sliderMax"
+      v-bind:aria-valuenow="faderRightValue">
+  </div>
+</template>
+
+<script>
+  import { library } from '@fortawesome/fontawesome-svg-core'
+  import { faKeyboard } from '@fortawesome/free-solid-svg-icons'
+  import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+
+  import Colored from '@/mixins/colored'
+  import Indicating from '@/mixins/indicating'
+  import Joined from '@/mixins/joined'
```

Comment:
Mixin `Joined`, `Rounded`, `Indicating`, `Invertible`, 'Simplified` etc are not being used.